index.bs

```diff
@@ -2932,6 +2924,15 @@ nothrow>WritableStreamFinishPendingCloseWithError ( <var>stream</var>, <var>reas
   1. Perform ! WritableStreamRejectClosedPromiseIfAny(_stream_).
 </emu-alg>
 
+<h4 id="writable-stream-is-ready-promise-pending" aoid="WritableStreamIsReadyPromisePending"
+nothrow>WritableStreamIsReadyPromisePending ( <var>stream</var>, <var>state</var>, <var>wasAborted</var> )</h4>
```

Comment:
I guess this is a preexisting problem, but I am not sure this is the best name. I would expect "ReadyPromiseIsPending" to just return whether stream.[[writer]].[[readyPromise]].[[PromiseState]] is "pending". This instead does a complex calculation on the states and the backpressure. What is the actual thing we are trying to capture here? Maybe "WritableStreamIsReadyForWrites"?